src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -33,14 +33,17 @@ public OrderController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
-        ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
+    public OrderResponseDto completeOrder(Authentication authentication) {
+        ShoppingCart cart =
+                shoppingCartService.getByUser(userService.findByEmail(
+                        authentication.getPrincipal().toString()).get());
```

Comment:
```suggestion
                shoppingCartService.getByUser(userService.findByEmail(
                        authentication.getName());
```
fix all similar cases